docs/source/changelog.rst

```diff
@@ -20,9 +20,10 @@ Changelog
         * Fix broken downloads badge in README.md (:pr:`1107`)
     * Testing Changes
         * Use CircleCI matrix jobs in config to trigger multiple runs of same job with different parameters (:pr:`1105`)
+        * Change to normalize from base entity with no time index for test_normalize_time_index_from_none (:pr:`1113`)
```

Comment:
actually this is in the wrong place. It needs to be above under Future Release. 0.18.1 is already released.

Reply:
Maybe change the description to `Better test case for normalizing from no time index to time index (:pr:`1113`)`

Reply:
made both of these changes